Gordon360/Controllers/ProfilesController.cs

```diff
@@ -89,16 +89,32 @@ public ProfilesController(IProfileService profileService, IAccountService accoun
                 faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
                 alumni = _alumni == null ? null : (PublicAlumniProfileViewModel)_alumni;
             }
+            else if (viewerGroups.Contains(AuthGroup.Alumni) && viewerGroups.Contains(AuthGroup.Student))
+            {
```

Comment:
If I'm not mistaken @EjPlatzer this shouldn't be possible?

Reply:
No this absolutely is possible.

Reply:
:monocle_face:

Reply:
This seems wired, but as Dr. Tuck told me, if a student is a senior, then s/he is both a student and a alumni.